### PR TITLE
Prs13307 adrian

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: new/PRS13307Adrian.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sameAs="Q374822" sex="1">
+                    <persName xml:lang="gez" xml:id="n1">እንድርያኖስ፡</persName>
+                    <persName xml:lang="gez" corresp="#n1">ʾƎndǝryānos</persName>
+                    <faith type="Christianity"/>
+                    <nationality type="Greece"/>
+                    <birth/>
+                    <death>306</death>
```
Comment (eu-genia) [inline new/PRS13307Adrian.xml:59]: diesNatalis for martyrs and saints is very helpful (see e.g. PRS8886Sisinios)

File: new/PRS13307Adrian.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <revisionDesc>
+            <change who="MKr" when="2020-12-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sameAs="Q374822" sex="1">
+                    <persName xml:lang="gez" xml:id="n1">እንድርያኖስ፡</persName>
+                    <persName xml:lang="gez" corresp="#n1">ʾƎndǝryānos</persName>
```
Comment (eu-genia) [inline new/PRS13307Adrian.xml]: Persons who are not Ethiopian should go under the well-known name, so you need the name "Adrian of Nicomedia" with the type="main" which will be printed as the record title

(check other non-Ethiopian saints/writers, e.g. PRS5720JohnChr, for example)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13307Adrian.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply new/PRS13307Adrian.xml:59]: But there is "diesNatalis" in the <note>. Do you want me to move it to the <floruit>?

Comment (eu-genia) [reply new/PRS13307Adrian.xml:59]: for some reason I could only see the empty floruit element in the PR in the pull request

Comment (eu-genia) [reply new/PRS13307Adrian.xml:59]: now I see it in the update, sorry I missed it, all OK! sorry

**Review by eu-genia — APPROVED** [review]

**Pull request merged**